drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilities.java

```diff
@@ -142,7 +143,7 @@ public static boolean canAcceptOtherwiseValues(final BaseColumn column) {
     private String[] parseValueList(String fieldType,
                                     String valueList) {
 
-        final String[] values = valueList.split(",");
+        final String[] values = ListSplitter.split(valueList);
```

Comment:
Could we test this change in `org.drools.workbench.screens.guided.dtable.client.widget.table.utilities.ColumnUtilitiesGetValueListTest` class please?

Reply:
Tried with a break point and this does get hit with several different setups where the character , is in the string.